launcher/main.cpp

```diff
@@ -58,11 +58,6 @@ int main(int argc, char* argv[])
     return 42;
 #endif
 
-#if QT_VERSION <= QT_VERSION_CHECK(6, 0, 0)
-    QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
-    QGuiApplication::setAttribute(Qt::AA_UseHighDpiPixmaps);
-#endif
```

Comment:
I have a feeling this condition is meant to be `QT_VERSION < QT_VERSION_CHECK(6, 0, 0)`...but this could break support for the first release maybe?